style.css

```diff
@@ -475,10 +473,344 @@ ul {
+    color: #6070ff;
+  }
+
+  .tools > :nth-child(1) {
+    margin-left: 0;
+  }
+
+  .button {
+    grid-area: p-button;
+    width: 100px;
+    height: 24px;
+    text-decoration: none;
+    color: #396df2;
+    padding: 10px;
+    border-radius: 10px;
+    text-transform: capitalize;
+    border: 1px solid #396df2;
+  }
+
+  .button:hover {
+    color: white;
+    background-color: #6070ff;
+  }
+
+  .button:active {
+    background-color: #2230d2;
+    color: white;
+  }
+
+  .buton:disabled {
+    color: #5e6c84;
+    border-color: #5e6c84;
+  }
+
+  /******************  about section  *********************/
+  .about {
+    display: grid;
+    grid-template-columns: 55% 45%;
+    grid-template-areas: 'about-self''tech';
+    padding: 150px 0;
+    margin: 0;
+  }
+
+  .wrapper-about {
+    grid-area: about-self;
+    background-color: white;
+    display: grid;
+    grid-template-rows: 100px 150px 60px 20px 30px;
+    grid-template-areas:
+      'about-title'
+      'about-subtitle'
+      'conect'
+      'about-social'
+      'about-button';
+    width: 80%;
+    height: 300px;
+    margin-left: 100px;
+  }
+
+  .about-title {
+    grid-area: about-title;
+    color: #172b4d;
+    width: 50%;
+    font-size: 45px;
+    font-weight: 700;
+    line-height: 52px;
+    margin-top: 0;
+  }
+
+  .about-subtitle {
+    grid-area: about-subtitle;
+    font-style: normal;
+    font-weight: normal;
+    text-align: left;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .conect {
+    grid-area: conect;
+    padding: 0;
+    display: grid;
+    align-content: center;
+    font-weight: normal;
+    font-size: 16px;
+  }
+
+  .about-social {
+    grid-area: about-social;
+    display: flex;
+    width: 20%;
+    justify-content: space-evenly;
+    align-content: center;
+  }
+
+  .about-button {
+    grid-area: about-button;
+    display: flex;
+    width: 140px;
+    font-size: 16px;
+    font-weight: 600;
+  }
+
+  .title > span {
+    display: flex;
+    align-self: center;
+    justify-content: flex-end;
+  }
+
+  .child-ul {
+    display: grid;
+    width: 70%;
+    height: 100%;
+    grid-template-columns: 30% 30% 30%;
+    align-content: center;
+    padding: 0;
+    gap: 10px;
+  }
+
+  /**********  contact-form  *************/
+  .contact-form {
+    display: flex;
+    flex-direction: column;
+    background: #6070ff;
+    padding: 103px 24px;
+    border-radius: 80px 0 0 0;
+  }
+
+  .contact-form > form {
+    height: 390px;
+    width: 40%;
+    margin: auto;
+    margin-top: 0;
+    gap: 30px;
+  }
+
+  .contact-subtitle {
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    padding-bottom: 20px;
+    width: 52%;
+    margin: auto;
+  }
+
+  .contact-button {
+    margin: auto;
+  }
+
+  .shapes-form {
+    position: absolute;
+    background-image: url('elements/shapes-form-desktop.png');
+    width: 993px;
+    height: 550px;
+    left: 0;
+    top: 3900px;
+  }
 }
```

Comment:
Ensure you stick to the design template as much as possible, please compare the contact section and make the necessary changes 🙏. Yours:
![Screenshot (48)](https://user-images.githubusercontent.com/52497246/121629036-4fde8e00-ca72-11eb-82ba-8db983cf46c8.png)

Original design:
![Screenshot (49)](https://user-images.githubusercontent.com/52497246/121629181-8ddbb200-ca72-11eb-975b-fff64d7c0905.png)